.github/workflows/ci.yml

```diff
@@ -33,23 +33,28 @@ jobs:
           java-version: '20'
 
       - name: Build and run checks
-        id: gradle-build
         uses: gradle/gradle-build-action@v2
```

Comment:
The Gradle Build Action docs recommend using the this 'global' caching, rather than invoking Gradle through the actions (see [here](https://github.com/gradle/gradle-build-action)). Happy to revert this if there's a reason it is done this way.